api/src/com/cloud/deploy/DeploymentClusterPlanner.java

```diff
@@ -55,7 +55,7 @@
             "true",
             "Enable/Disable cluster thresholds. If disabled, an instance can start in a cluster even though the threshold may be crossed.",
             false,
-            ConfigKey.Scope.Zone);
+            ConfigKey.Scope.Global);
```

Comment:
This refers to cluster level setting, admins should be allowed to override this at the cluster level. Please change this to ConfigKey.Scope.Cluster.